elements/bulbs-quiz/bulbs-quiz.scss

```diff
@@ -0,0 +1,3 @@
+bulbs-quiz {
+  display: block;
```

Comment:
I don't think it is in the case of custom elements. @collin can you confirm this?